config/default_records/submission_templates/001_cardinal_submission_templates.wip.yml

```diff
@@ -3,13 +3,15 @@
 # the real release.
 ---
 Limber - Cardinal:
+  name: "limber_cardinal"
```

Comment:
I think maybe these 'names' shouldn't be here - perhaps a hangover from the issue where the submission template records were being created with 'limber_cardinal' as the name rather than 'Limber - Cardinal' as the name? I think you changed your local database rather than doing the fix like this - https://github.com/sanger/sequencescape/pull/3411/files